apps/nmreplay/nmreplay.c

```diff
@@ -972,7 +972,7 @@ usage(void)
 {
 	fprintf(stderr,
 	    "usage: nmreplay [-v] [-D delay] [-B {[constant,]bps|ether,bps|real,speedup}] [-L loss]\n"
-	    "\t[-b burst] -f pcap-file -i ifb\n");
+	    "\t[-b burst] -f pcap-file -i netmap:<iface_name>\n");
```

Comment:
actually this accepts any interface name that can be passed to nm_open (see `man netmap`).
Examples:
netmap:eth0
netmap:eth0/T
netmap:eth0-3
vale3:x
netmap:eth1^
valex:y{34
...

Reply:
PR updated - let me know what you think.